tags/app.tag

```diff
@@ -1,13 +1,20 @@
 <app>
   <nav><a each={ data } href="#{ id }"> { title }</a></nav>
 
+  <p>
+    <a href="#/">Home</a> |
+    <a href="#/1">create app</a> |
+    <a href="#/2">app detail</a>
+  </p>
+
   <article>
-    <h1>{ page.title || 'Not found' }</h1>
-    <p>{ page.body|| 'Specified is not found' }</p>
+    <h1>{ page.title }</h1>
+    <p>{ page.body }</p>
```

Comment:
The error you have seen was because of the `||` within the `{}` placeholders. Which is weird, I agree that this should work. Might be worth creating an issue on the riotjs repository for it